src/test/java/net/sf/jabref/logic/cleanup/CleanupWorkerTest.java

```diff
@@ -167,6 +168,36 @@ public void cleanupDoiReturnsChangeWhenDoiInURLField() {
     }
 
     @Test
+    public void cleanupISSNReturnsCorrectISSN() {
```

Comment:
I would move these tests to a new ISSNCleanupTests class. Don't know why nearly all cleanup tests are bundled here in this class.